pkg/clusterd/network_test.go

```diff
@@ -1,3 +1,18 @@
+/*
```

Comment:
were you able to search for any other code files that are missing the copyright header?

Reply:
i stumbled onto this one. Searching for something that is missing is hard :)